Common/Orders/OrderTicket.cs

```diff
@@ -273,6 +273,72 @@ public OrderResponse Update(UpdateOrderFields fields)
             return _updateRequests.Last().Response;
         }
 
+        /// <summary>
+        /// Submits an <see cref="UpdateOrderRequest"/> with the <see cref="SecurityTransactionManager"/> to update
+        /// the ticket with tag specified in <paramref name="tag"/>
+        /// </summary>
+        /// <param name="tag"></param>
+        /// <returns></returns>
+        public OrderResponse UpdateTag(string tag)
+        {
+            UpdateOrderFields fields = new UpdateOrderFields();
```

Comment:
Please use `var` and inline construction here. Please complete the xml params/results documentation.
```
var fields = new UpdateOrderFields() 
{
        Tag = tag
};
```

Reply:
Changed.